src/engine/server/server.cpp

```diff
@@ -4218,7 +4218,20 @@ class CSqlJob_Server_SendPlayerStatistics : public CSqlJob
 	void UpdateScore(CSqlServer* pSqlServer, int ScoreType, int Score, const char* pScoreName)
 	{
 		char aBuf[512];
-				
+		
+		int PlayerCount = 0;
+		for(int i=0; i<MAX_CLIENTS; i++)
+		{
+			if(m_apPlayers[i] && m_apPlayers[i]->m_IsInGame == true)
+			{
+				if (std::find(vec.begin(), vec.end(), i) == vec.end())
```

Comment:
sorry, but what is `vec` here? What does this line do?

Reply:
> vec contains all spectators.

And where is `vec` defined and initialized?

Reply:
even worse, if `vec` is not defined, how does it even compile?

Reply:
CServer::GetActivePlayerCount() provides access to active players, there is no need to access this vec. And yes it was not defined before. It is defined in GetActivePlayerCount()

If (GetActivePlayerCount() > ... ) then do smth

Reply:
yeah forgot to use `server_sql_release` as conf. got it working with a sql database and tested it on oi2 labs